plugins/modules/sonic_vrfs.py

```diff
@@ -16,8 +16,9 @@
 module: sonic_vrfs
 version_added: 1.0.0
 notes:
-- Tested against Enterprise SONiC Distribution by Dell Technologies.
-- Supports C(check_mode).
+  - Tested against Enterprise SONiC Distribution by Dell Technologies.
+  - Supports C(check_mode).
+  - This module does not support the deletion of mgmt VRF. Deletion of mgmt VRF can be done using the "sonic_config" resource module as shown in "playbooks/common_examples/mgmt_vrf_off.yaml". It can also be done using the SONiC CLI or by using a standalone REST API.
```

Comment:
```suggestion
  - This module does not support the deletion of mgmt VRF. Deletion of mgmt VRF can be done using the
    "sonic_config" resource module as shown in "playbooks/common_examples/mgmt_vrf_off.yaml". It can also
     be done using the SONiC CLI or by using a standalone REST API.
```